src/parser/jobs/whm/modules/Lilies.tsx

```diff
@@ -57,6 +67,8 @@ export class Lilies extends CoreGauge {
 
 	@dependency private suggestions!: Suggestions
 
+  private numUnnecessaryGcdHeals = 0;
```

Comment:
Prefer to name these by what they do instead of a verbose abbreviation. Something like `wastedGcds` is more than clear enough. Would need to update uses of the var too.